### DISCUSSION TITLE
Is there a low-javascript version of web Github?

### DISCUSSION THREAD

**Discussion by JamesC1** [community/community#53675] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I'm on a memory-constrained machine, and old-fashioned websites are much better for me than fancy ones.  eg I use html-only gmail.

**Comment by scottajames** [comment]
Remember reading about GitHub Lite that was suppose to be a reduced JavaScript version but I think they stopped it in 2021, you could disable Javascript yourself though in your settings and see if that works?
